src/archanaNJun20/PalindromeString.java

```diff
@@ -0,0 +1,23 @@
+package archanaNJun20;
+
+public class PalindromeString {
+	void palindromeString(String inputString){ 
+		String originalstring=inputString;
+		int length=inputString.length();
+		String newString = "";
+		for(int index=length-1;index>=0;index--){
+			char ch=inputString.charAt(index);
+			newString=newString+String.valueOf(ch);
```

Comment:
you can also write like this newString += String.valueOf(inputString.charAt(index));

Reply:
The code is updated based on the comment. Thank you for your Suggestion.